docs/tokens/sol-token.mdx

```diff
@@ -15,19 +15,21 @@ import LegacyContentBanner from '@site/src/theme/LegacyContentBanner'
 The Helium Network runs on the Solana blockchain, an extremely fast, scalable, and low-cost
 blockchain.
 
-Interacting with the Solana blockchain requires a small fee (average transaction fee is 0.000014
-SOL) paid in SOL tokens, which is massively cheaper than the original Helium blockchain transaction
+Interacting with the Solana blockchain requires a small fee (the average transaction fee is 0.000117590
+SOL*) paid in SOL tokens, which is massively cheaper than the original Helium Blockchain transaction
```

Comment:
let's use the footnote syntax. Here is some reference: https://github.com/helium/docs/blob/650560ae107e198028ae53457acac5c6ed405489/docs/tokens/iot-token.mdx#L15-L21

Love the cited source.

Reply:
Sure, no problem.